R/deployments.R

```diff
@@ -79,9 +80,11 @@ deployments <- function(appPath = ".",
 
 deploymentFields <- c(
   "name", "title", "username", "account", "server", "hostUrl", "appId",
-  "bundleId", "url", "envVars"
+  "bundleId", "url", "envVars", "version"
 )
 
+deploymentVersion <- 1L
```

Comment:
Maybe this should be `deploymentRecordVersion`?

Reply:
I like that better.